config/environment.default.json

```diff
@@ -0,0 +1,26 @@
+{
+  "production": false,
+  // Location of REST API
```

Comment:
Unfortunately, comments are not allowed in a json file. They're not part of [the spec](http://www.json.org/)